mainnet/pregenesis.json

```diff
@@ -0,0 +1,369 @@
+        "denom": "stake",
+        "amount": "0"
+      },
+      // stargaze
+     "module_account_balance": {
+        "denom": "ustarx",
+        "amount": "197999999959202"
+      },
+      "params": {
+        "airdrop_start_time": "0001-01-01T00:00:00Z",
+        "duration_until_decay": "3600s",
+        "duration_of_decay": "18000s",
+        "claim_denom": "ustarx"
+      },
+      // stargaze
+      "params": {
+        "airdrop_start_time": "2022-07-29T17:00:00Z",
+        "duration_until_decay": "432000s",
+        "duration_of_decay": "432000s",
+        "claim_denom": "ustarx"
+      },
+    },
+    // default
+    "crisis": {
+      "constant_fee": {
+        "denom": "ustarx",
+        "amount": "1000"
+      }
+    },
+    // hub-4
+    "crisis": {
+      "constant_fee": {
+        "amount": "1333000000",
+        "denom": "uatom"
+      }
+    },
+    // stargaze
+    "crisis": {
+      "constant_fee": {
+        "denom": "ustarx",
+        "amount": "500000000000"
+      }
+    },
+    "distribution": {
+      "params": {
+        "community_tax": "0.020000000000000000",
+        "base_proposer_reward": "0.010000000000000000",
+        "bonus_proposer_reward": "0.040000000000000000",
+        "withdraw_addr_enabled": true
+      },
+    // stargaze
+    "distribution": {
+      "params": {
+        "community_tax": "0.050000000000000000",
+        "base_proposer_reward": "0.010000000000000000",
+        "bonus_proposer_reward": "0.040000000000000000",
+        "withdraw_addr_enabled": true
+      },
+    },
+    "gov": {
+      "starting_proposal_id": "1",
+      "deposits": [],
+      "votes": [],
+      "proposals": [],
+      // default
+      "deposit_params": {
+        "min_deposit": [
+          {
+            "denom": "ustarx",
+            "amount": "10000000"
+          }
+        ],
+        "max_deposit_period": "172800s"
+      },
+      // hub-4
+      "deposit_params": {
+        "min_deposit": [
+          {
+            "amount": "512000000",
+            "denom": "uatom"
+          }
+        ],
+        "max_deposit_period": "1209600s"
+      },
+      // stargaze
+     "deposit_params": {
+        "min_deposit": [
+          {
+            "denom": "ustarx",
+            "amount": "1000000"
+          }
+        ],
+        "max_deposit_period": "1209600s"
+      },
+      "voting_params": {
+        "voting_period": "172800s"
+      },
+      // stargaze
+      "voting_params": {
+        "voting_period": "86400s"
+      },
+      // default
+      "tally_params": {
+        "quorum": "0.334000000000000000",
+        "threshold": "0.500000000000000000",
+        "veto_threshold": "0.334000000000000000"
+      }
+     // hub-4
+      "tally_params": {
+        "quorum": "0.400000000000000000",
+        "threshold": "0.500000000000000000",
+        "veto_threshold": "0.334000000000000000"
+      },
+    // stargaze
+     "tally_params": {
+        "quorum": "0.100000000000000000",
```

Comment:
For mainnet, I feel this should be a bit higher. 20%? 25%?

Reply:
20% sounds good to me (since 20% will be locked up the claims module, don't want to make this too high).